### PR TITLE
Integrate hooks with the rest translation server

### PR DESCRIPTION
**Pull request by panosk**

### CODE REVIEW

File: tools/rest_translation_server.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ]]
 
 require('onmt.init')
-
+require('copas')
```
Comment (jsenellart) [inline tools/rest_translation_server.lua]: this does not belong to this PR, is this?
do you have an integration of copas in the rest server?

Comment (panosk) [reply tools/rest_translation_server.lua]: Hi @jsenellart ,

Actually, I just blindly included it after getting the error: 
`you must require copas before require'ing socket.http`
Without this require, the server won't start with hooks.

Comment (panosk) [reply tools/rest_translation_server.lua]: On a second glance, this requirement seems to occur because of the tree-tagger.lua hook which requires `httpclient`. Maybe we should just move the copas require to the tree-tagger.lua?

Comment (jsenellart) [reply tools/rest_translation_server.lua]: yes - it will make sense. Also, I will also independently look at integrating with copas in the servers.
